petpals/messaging/static/js/messages.js

```diff
@@ -0,0 +1,6 @@
+window.onload = function() {
+    messageItems = document.querySelectorAll(".message-card");
+    if (messageItems.length == 0) {
```

Comment:
Shouldn't be querySelector**All** and should be triple equals.
Or you can just add a `if not recent_messages` in jinja as it would be much less work.

Reply:
Agree with '===', don't think selecting one or many matters though.